--- config.template
+++ config.default
@@ -15,5 +15,5 @@
 #BACKEND = mixer_sdl
 #BACKEND = mixer_sdl2
-#BACKEND = posix
+BACKEND = posix

 #OSTYPE = amiga
@@ -22,5 +22,5 @@
 #OSTYPE = freebsd
 #OSTYPE = haiku
-#OSTYPE = linux
+OSTYPE = linux
 #OSTYPE = mingw
 #OSTYPE = mac
@@ -28,5 +28,5 @@
 #DEBUG = 1    # Level 1-3, higher number means more debug-friendly but slower, see Makefile
 #MSG_LEVEL = 1 # Level 1-4, more runtime debug messages (without only warnings and errors)
-#OPTIMISE = 1 # Add umpteen optimisation flags
+OPTIMISE = 3 # Add umpteen optimisation flags
 #PROFILE = 1  # Enable profiling
 #PROFILE = 2  # Enable profiling with optimisation flags, can be used with `OPTIMISE = 1'
@@ -40,5 +40,5 @@
 #WIN32_CONSOLE = 1 # adds a console for windows debugging

-#MULTI_THREAD = 1 # Enable multithreading
+MULTI_THREAD = 1 # Enable multithreading

 # using freetype for Truetype font support
@@ -46,8 +46,8 @@

 # using UPnP for easy server hosting behind routers
-#USE_UPNP = 0
+USE_UPNP = 1

 # using zstd compression library
-#USE_ZSTD = 0
+USE_ZSTD = 1

 # Define these as empty strings, if you don't have the respective config program
